### DISCUSSION TITLE
IoTDB调用RESTful服务，使用ping接口显示Empty reply from server

### DISCUSSION THREAD

**Discussion by peanut0611** [apache/iotdb#12037] [Q&A]
![image](https://github.com/apache/iotdb/assets/136685092/18e17728-22f1-46a1-a0cd-8ffb71670123)

![image](https://github.com/apache/iotdb/assets/136685092/56fead4f-eaf6-4c9d-b6d2-b043b7276823)

**Comment by HTHou** [comment]
6667 端口是iotdb 默认的客户端thrift 服务端口，不是rest服务端口，默认的rest服务端口是文档中的18080，并请确保iotdb的rest服务开关已开启。
